server/controllers/amp-page.js

```diff
@@ -71,6 +71,7 @@ function getAndRender(uuid, options) {
 
 			const shareParams = {};
 			article.shareUrl = `${article.webUrl}?${querystring.stringify(shareParams)}`;
+			article.facebookAppId = '121355018262104';
```

Comment:
Oh man. Now's the time to admit I really don't understand what a Facebook app *is*.